.tools/ailly/docs/10_scout_how_to.md

```diff
@@ -0,0 +1,285 @@
+---
+skip: True
+prompt: Write instructions on how to do Workflow Scouting with Ailly.
+---
+
+# How to Scout with Ailly
+
+## Prereq: Install and run Ailly
+
+1. On bash, easiest way is probably an alias: `alias ailly=npx @ailly/cli`
+   - which will probably need `npm install -g @ailly/cli`
```

Comment:
If you're doing a global install anyway, the alias probably isn't necessary. I'm not using npx anymore.